protobuf/syft_proto/execution/v1/communication_action.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package syft_proto.execution.v1;
+option java_package = "org.openmined.syftproto.execution.v1";
+
+import "syft_proto/types/syft/v1/arg.proto";
+import "syft_proto/types/syft/v1/id.proto";
+
+message CommunicationAction {
+    syft_proto.types.syft.v1.Arg obj = 1;
```

Comment:
@Jasopaum This looks kinda like `ObjectMessage`; I think we want something a bit different here. I was imagining the `CommunicationAction` message would describe the action of sending an object, so that we can send that action to another worker as a command (and then it can send the object.)

If Alice says to Bob "hey, send Tensor1 to Charlie", the `CommunicationAction` would be "send Tensor1", which would result in Bob sending an `ObjectMessage` containing Tensor1. Sound reasonable?

Reply:
Hum, I think that's what I wanted to do...
The communication action in your example would look like (obj: Tensor1, source: Bob, destinations: [Charlie], kwargs: {}).

Reply:
Ah, okay! In that case, I think the type should probably be `syft_proto.types.syft.v1.id`, since we want to send a reference to the object (instead of the object itself.)

Reply:
Right, it'll be better with an id!